boards/hifive1/include/periph_conf.h

```diff
@@ -30,27 +30,37 @@ extern "C" {
  * @name    Core Clock configuration
  * @{
  */
-#define USE_CLOCK_HFXOSC_PLL        (1)
-#define USE_CLOCK_HFXOSC            (0)
-#define USE_CLOCK_HFROSC_PLL        (0)
+#ifndef CONFIG_USE_CLOCK_HFXOSC_PLL
+#define CONFIG_USE_CLOCK_HFXOSC_PLL         (1)     /* Use PLL clocked by HFXOSC by default */
+#endif
+#ifndef CONFIG_USE_CLOCK_HFXOSC
+#define CONFIG_USE_CLOCK_HFXOSC             (0)
+#endif
+#ifndef CONFIG_USE_CLOCK_HFROSC_PLL
+#define CONFIG_USE_CLOCK_HFROSC_PLL         (0)
+#endif
```

Comment:
The problem with this is the way Kconfig handles boolean symbols, when a `bool` symbol is `n` it does not get the value `0`, it's just not defined. So, doing this prevents from ever disabling `CONFIG_USE_CLOCK_HFXOSC_PLL` from Kconfig.

For instance in this case we had to invert the logic of the symbol to have a better default
https://github.com/RIOT-OS/RIOT/blob/42eb044ec67758e1664dd354cf07652c9632aace/sys/include/net/gnrc/sixlowpan/config.h#L120-L126